app/src/main/graphql/checkout.graphql

```diff
@@ -78,8 +78,8 @@ mutation CompleteOnSessionCheckout($checkoutId: ID!, $paymentIntentClientSecret:
   }
 }
 
-mutation completeOrder($projectId: ID! $stripePaymentMethodId: String, $paymentSourceId: String, $paymentSourceReusable: Boolean, $paymentMethodTypes: [String!]) {
-  completeOrder(input:{ projectId: $projectId, stripePaymentMethodId: $stripePaymentMethodId, paymentSourceId: $paymentSourceId, paymentSourceReusable: $paymentSourceReusable, paymentMethodTypes: $paymentMethodTypes }) {
```

Comment:
@mtgriego im a bit nervous about this change. this change was required after i downloaded the new graphql schema.

Reply:
This would affect the feature branch, but if changes are needed there we can add it back in before merging to master, though to be safe lets hold the merge until we can confirm with Isa tomorrow